resources/demos/angular-imports.js

```diff
@@ -1,4 +1,5 @@
 // Export generic functions.
+// eslint-disable-next-line import/extensions
```

Comment:
I recommend, instead of ignoring this line, to delete the `.js` extension of the import so the lint does not give any more errors. The same applies to the `resources/demos/imports.js` and `resources/demos/react-imports.js` files.

Reply:
Be my guest.
I did not dare to change it since it only affecte the 'demos' and didn't consider wothwhile to dedicate much more effort on this, honestly.

Reply:
Well, maybe we can let it like this for now, and change it in the future since that isn't actually in this scone either